dips/dip-183.md

```diff
@@ -0,0 +1,103 @@
+---
+dip: 183
+title: Sender Initiated Reference ID Exchange
+authors: Sunmi Lee (@sunmilee), David Wolinsky (@davidiw)
+status: Draft
+type: Informational
+created: 06/08/2021
+last_updated: 06/08/2021
+issue: https://github.com/diem/dip/issues/183
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# Summary
+
+This DIP describes a protocol standard for sender initialized pre-flight exchange using reference IDs to form payment transactions.
+
+A reference ID is used as a unique identifier for a payment transaction between two parties across two VASPs without revealing any information about those parties on-chain.
+In this protocol, the sender already knows the recipient VASP address and an identifier for the recipient at their VASP. The sender VASP contacts the recipient VASP and shares the sender's identifier, a reference ID, and the recipient's identifier. 
+Upon conclusion, the sender VASP transmits to the blockchain a transaction with the reference ID, completing the protocol without revealing any potentially identifiable user information publicly on-chain.
+
+Reference IDs are unique between a pair of VASPs, `VASP A` and `VASP B`,  so if a reference ID `r` was used in a previous transaction between the two VASPs, regardless of which one initiated it, `r` cannot be reused for any future transactions between `VASP A` and `VASP B`. 
+However, the same `r` value can be used between a different VASP pair, `VASP A` and `VASP C`. VASPs may choose to keep reference IDs globally unique across all VASPs as well. 
```

Comment:
Also added more details on ref ID uniqueness @aching

Reply:
Personally I think the first sentence was good enough, but I'd rather over specify than have confusion :P